br/backup-and-restore-overview.md

```diff
@@ -1,114 +1,135 @@
+- BR uses resources of the target cluster as much as possible. Therefore, it is recommended that you restore data to a new cluster or an offline cluster. Avoid restoring data to a production cluster. Otherwise, your application might be affected. PITR only supports restoring data to an empty cluster.
 
-#### Some tips
+PITR:
 
-The following are some recommended operations for using BR:
+- You can only perform cluster-level PITR. Database-level and table-level PITR are not supported.
+- You cannot restore data in the user tables or the privilege tables.
 
-- It is recommended that you perform the backup operation during off-peak hours to minimize the impact on applications.
-- BR only supports restoring data to a new cluster and uses resources of the target cluster as much as possible. Therefore, it is not recommended that you restore data to a production cluster. Otherwise, services might be affected.
-- It is recommended that you execute multiple backup or restoration operations one by one. Running backup or restoration operations in parallel reduces performance and also affects online applications. Worse still, lack of collaboration between multiple tasks might result in task failures and affect cluster performance.
-- Amazon S3, Google Cloud Storage, and Azure Blob Storage are recommended to store backup data.
-- Make sure that the BR and TiKV nodes, and the backup storage system have sufficient network bandwidth to ensure sound write/read performance. Insufficient storage capacity might be the bottleneck for a backup or restoration operation.
-
-### See also
-
-- [Back up Data to S3-Compatible Storage Using BR](https://docs.pingcap.com/tidb-in-kubernetes/stable/backup-to-aws-s3-using-br)
-- [Restore Data from S3-Compatible Storage Using BR](https://docs.pingcap.com/tidb-in-kubernetes/stable/restore-from-aws-s3-using-br)
-- [Back up Data to GCS Using BR](https://docs.pingcap.com/tidb-in-kubernetes/stable/backup-to-gcs-using-br)
-- [Restore Data from GCS Using BR](https://docs.pingcap.com/tidb-in-kubernetes/stable/restore-from-gcs-using-br)
-- [Back up Data to PV](https://docs.pingcap.com/tidb-in-kubernetes/stable/backup-to-pv-using-br)
-- [Restore Data from PV](https://docs.pingcap.com/tidb-in-kubernetes/stable/restore-from-pv-using-br)
+Backup storage and network configuration:
+
+- It is recommended that you store backup data to a storage system that is compatible with Amazon S3, GCS, or Azure Blob Storage.
+- You need to ensure that the br command-line tool, TiKV, and the backup storage system have enough network bandwidth, and the storage system can provide sufficient read and write performance (IOPS). Otherwise, they might become a performance bottleneck during backup and restore.
+
+### Compatibility
+
+Before you use BR feature of TiDB, pay attention to the usage restrictions, compatibility, and other considerations of BR.
```

Comment:
```suggestion
```

Reply:
There should be a short description between different levels of titles. If you think the content inappropriate, you can polish it.